### PR TITLE
Added responsiveness for all the screens

### PR DESCRIPTION
**Pull request by Razen04**
I have added responsiveness for all the screen size and all fixed the evaluation bar which was not working before for mobile screen sizes.

### CODE REVIEW

File: nodemon.json
Comment (WintrCat) [inline nodemon.json:1]: remove public from the watch list, it is inside src so changes to public will be watched anyway

File: src/public/global.css
Comment (WintrCat) [inline src/public/global.css:1]: code smell: there should be a space between CSS properties and values

File: src/public/pages/report/styles/index.css
Comment (WintrCat) [inline src/public/pages/report/styles/index.css:1]: code smells: there should always be a space between the CSS selector and the `{`.

File: src/public/pages/report/styles/index.css [hunk 1]
```diff
@@ -1,224 +1,170 @@
...
-
-@media (min-width: 1000px) and (min-height: 860px) {
-
-    #review-container {
-        display: grid;
-        
-        grid-template-columns: 68vw 28vw;
-        grid-template-rows: 90vh;
-    }
+@media only screen and (max-width: 1536px){
```
Comment (WintrCat) [inline src/public/pages/report/styles/index.css:7]: here is an example of said code smell

File: src/public/pages/report/styles/index.css [hunk 2]
```diff
@@ -1,224 +1,170 @@
...
-        flex-direction: column;
+        height: 610px !important;
     }
 
-    #secondary-message {
-        width: 60%;
-    }
-    #kofi-a, #discord-a{
-        padding: 20px !important;
+    #classification-message-container{
```
Comment (WintrCat) [inline src/public/pages/report/styles/index.css:22]: and a second example

File: src/public/pages/report/styles/reviewpanel.css
Comment (WintrCat) [inline src/public/pages/report/styles/reviewpanel.css:1]: code smells: there should always be a space between the CSS selector and the `{`.

File: src/public/pages/report/styles/reviewpanel.css [hunk 3]
```diff
@@ -256,46 +252,61 @@
...
     margin-top: 7px;
-    justify-content:flex-start;
+    justify-content: flex-start;
     width: 100%;
     background-color: var(--primary-color);
     position: relative;
 }
-.badboystiff{
+
+.depthTitle{
```
Comment (WintrCat) [inline src/public/pages/report/styles/reviewpanel.css:299]: an example of said code smell

**Review by WintrCat — CHANGES_REQUESTED** [review]
thanks for the PR :) i've given you a code review - there are some changes that need implementing before we can proceed with the merge.

Besides the comments in the code review, there is was additional bug I found - between screen widths of 430 through 960 pixels, the responsiveness of the board breaks down. This will need fixing before merging. I've attached a screenshot of it:
image

**Comment by Razen04** [comment]
I have fixed the said issues, you can check it.

**Comment by Razen04** [comment]
I also fixed a small problem of arrow functioning from Issue #37.

File: src/public/pages/report/scripts/loadgame.ts
Comment (WintrCat) [inline src/public/pages/report/scripts/loadgame.ts:1]: This code does not work - the dialog breaks down completely if the user switches pages to a different year.

**Review by WintrCat — CHANGES_REQUESTED** [review]
Thanks for your updates :)

The code for the game selection arrows breaks down as soon as the user flips to a page in a different year, which needs to be addressed. From a quick glance it looks like you need to update some of the if-statements. Code smells were resolved, although the evaluation bar is still distorted when the height of the board decreases.

image

**Comment by Razen04** [comment]
> the evaluation bar is still distorted when the height of the board decreases

Do you mean the edges and of bar and the number in it? 

For the edges I can just remove the border radius and for number I don't if it is can be fixed using CSS. If there is something else you mean you can let me know and I will fix it.

For the date I didn't see the year changing problem while checking so I will se if I can fix it or not.

File: src/public/pages/report/scripts/loadgame.ts [hunk 4]
```diff
@@ -190,7 +186,7 @@ function registerModalEvents() {
 
     $("#next-page-button").on("click", () => {
 
-        if (gamesPeriod.month == 12 && (gamesPeriod.year - 1) >= new Date().getFullYear()) {
+        if (gamesPeriod.month == 12 && (gamesPeriod.year) >= new Date().getFullYear()) {
```
Comment (WintrCat) [inline src/public/pages/report/scripts/loadgame.ts:189]: code smell: brackets around gamesPeriod.year are not needed when there is no subtraction

**Review by WintrCat — COMMENTED** [review]
Thanks for this :)

The game selection date issue was fixed with your commit, although as I have said we aren't able to merge anything without the evaluation number issue being fixed. It is still distorted on smaller screen widths. The evaluation bar itself is an HTML canvas, so I believe changing the height CSS property of it will distort its contents. You may have to change the height of the canvas internally, which means you may have to change the JS implementation such that it updates itself in accordance to the page width. I'm not 100% sure though, I would guess there has to be some cleaner way to get this done.

**Comment by Razen04** [comment]
Fixed the distortion using SVG as they are easy to scale down for smaller screens without any distortions. I had to change the eval bar script for this to work with SVG instead of HTML Canvas.

**Review by WintrCat — APPROVED** [review]
thank you :)

**Pull request merged**